app/src/main/res/drawable/file_doc.xml

```diff
@@ -18,7 +18,38 @@
 
   Icon provided by Android Material Library in Apache License 2.0
 -->
-<vector android:height="72dp" android:viewportHeight="16"
-    android:viewportWidth="16" android:width="72dp" xmlns:android="http://schemas.android.com/apk/res/android">
-    <path android:fillColor="#49abea" android:pathData="m2.5,1c-0.28,0 -0.5,0.22 -0.5,0.5v13c0,0.28 0.22,0.5 0.5,0.5h11c0.28,0 0.5,-0.22 0.5,-0.5v-10.5l-3,-3h-8.5zM4,3h6v1h-6v-1zM4,6h5v1h-5v-1zM4,9h8v1h-8v-1zM4,12h4v1h-4v-1z"/>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I assume there will be way more icons to be replaced (like notification icon, etc.)
Thus I think it is better to have them in one "icon" PR.

Reply:
Noted